### ISSUE TITLE
Comparing different quantities?

### ISSUE THREAD

**Issue by Nightblade** [VolatilePulse/PoB-Item-Tester#10]
(Apologies if this is hard to follow, I'm not well and I'm fighting a heap of brain-fog atm.)

Is this code is comparing different quantities of some stats?   For example, here's 3 lines from `mods.json` :  
```
"12% increased Cold Damage"
"Adds 14 to 31 Cold Damage"
"Damage Penetrates 8% Cold Resistance"
```
This is going to calculate incorrect weightings unless I'm missing something, which is quite possible.

This could be easily fixed by changing `mods.json` so that each mod is normalized to 1 unit.  Using the example above:
```
"1% increased Cold Damage"
"Adds 1 to 1 Cold Damage"
"Damage Penetrates 1% Cold Resistance"
```

**Comment by Nightblade** [comment]
Another, perhaps easier fix:

Replace this line:
```lua
    local dps = findModEffect(mod.desc, statField, actorType)
```
With this:
```lua
    local dps = findModEffect(mod.desc, statField, actorType) / mod.count
```

**Comment by Nightblade** [comment]
Upon further investigation, I think Xanthics' web page does something similar to normalize the stats.  If this is the case please close this issue!  :)

**Comment by coldino** [comment]
Yes, the website knows the quantities and makes the relevant adjustments.
The numbers in the mod lines are reasonable numbers for real mod lines in the wild. Reducing some of them down to 1% or 1 point would sometimes hide the real effect in the noise, or within float or rounding errors.
